fotogalleri/fotogalleri/settings.py

```diff
@@ -176,7 +176,7 @@
 THUMB_QUEUE_THREAD_COUNT = env('THUMB_QUEUE_THREAD_COUNT', 4)
 
 # Enable thumbnail generation
-ENABLE_THUMB_QUEUE = env('ENABLE_THUMB_QUEUE', False)
+ENABLE_THUMB_QUEUE = env('ENABLE_THUMB_QUEUE', True)
```

Comment:
This is the reason why the tests are not terminating.

The tests do not seem able to kill thumbnail generation by themselves. I propose we keep this as `False` and document it properly instead.